apps/examples/wifi_manager/wm_test/wm_test_client.c

```diff
@@ -0,0 +1,712 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
Can we please change the year and wherever applicable ?

Reply:
Changes done as suggested